Lib/glyphsLib/types.py

```diff
@@ -228,7 +228,12 @@ def read(self, src):
         """Parse a datetime object from a string."""
         # parse timezone ourselves, since %z is not always supported
         # see: http://bugs.python.org/issue6641
-        string, tz = src.rsplit(' ', 1)
+        srcs = src.split(' ')
```

Comment:
the variable name "srcs" is not very clear to me. Maybe something like "chunks" or "words" would make more sense?